### ISSUE TITLE
Issue with Flask API Path 

### ISSUE THREAD

**Issue by hsihak** [mag9073/CSCN72030-Sec3-Group1#5]
### Problem

Currently, i am experiencing a minor issue related to path errors and I am seeking for help if possible. My goal is to seperate the **Flask API** call from the frontend to the backend into its own Middleware folder to makie it more organized. However, when i tried to execute the **Flask Server** to call KNN module for Diabates Prediction which call the "diabetes.csv" file, it was not able to detect file. I suspect that i may have defined a wrong path. 

Below, i have attached a screenshot of the project layout and error message. If you would like to reproduce the error, please contact me directly on our Group 1 - MS Teams :) 

Cheers! 🍻

![image](https://github.com/mag9073/CSCN72030-Sec3-Group1/assets/92398417/90ca8b95-b6df-4dc2-8e7b-c69b4945d6c1)

**Comment by bhavaysavaliya** [comment]
Please add all files in the repository otherwise it wouldn't be possible to help you

**Comment by bhavaysavaliya** [comment]
I figured out the problem. You should write path as "../../backend/KNNModule/KNNDiabetes/diabetes.csv" because diabetes file is being accessed to a path which is in parent of parent directory.

**Comment by hsihak** [comment]
> I figured out the problem. You should write path as "../../backend/KNNModule/KNNDiabetes/diabetes.csv" because diabetes file is being accessed to a path which is in parent of parent directory.

Thank you so much, Bhavay! We really appreciate you helping us resolve the issue. Despite our efforts to implement your suggested idea, we are still facing a similar error in that flask does not recognize the specified path. 

Due to the time, our group has decided to revert back to our original implementation. We moved the `api.py` flask file back into the backend folder, placing it in the root folder that fixed the issue. Below are the changes I have made. 

![image](https://github.com/mag9073/CSCN72030-Sec3-Group1/assets/92398417/e8339a82-76f3-4606-a2c0-5379d43075df)


Once again, thank you for your effort in helping us address the issue. Happy Hacking! 🙌
